src/js/stores/search.js

```diff
@@ -227,7 +227,8 @@ define(function (require, exports, module) {
                         return items.concat(options);
                     }.bind(this), []);
 
-                search.searchItems = Immutable.List(searchItems).unshift(filterItems);
+                searchItems.push(Immutable.List(filterItems));
```

Comment:
Nope-- searchItems is an list of lists, and `filterItems` is one of the inner lists.